src/Common.cpp

```diff
@@ -19,13 +19,13 @@ using namespace NAS2D;
 std::string StringFromDifficulty(const Difficulty& difficulty)
 {
 	switch(difficulty)
-	{
+    {
     case Difficulty::Beginner:  return constants::DIFFICULTY_BEGINNER;
     case Difficulty::Easy:      return constants::DIFFICULTY_EASY;
     case Difficulty::Medium:    return constants::DIFFICULTY_MEDIUM;
     case Difficulty::Hard:      return constants::DIFFICULTY_HARD;
-	default:                    return constants::DIFFICULTY_EASY;
-	}
+    default:                    return constants::DIFFICULTY_EASY;
+    }
```

Comment:
Indentation still doesn't look right. The `switch` is using a tab, while the block contents are using spaces.

Reply:
Are the tabs for github set to 5?

Reply:
Point is, the whole file is using tabs, except for the indented block of `case` statements.

Reply:
Though it would be ingenious if GitHub used a non-standard value for tabs, just to make the differences stand out more.